charts/bulk-scan-orchestrator/values.yaml

```diff
@@ -136,6 +139,12 @@ java:
           alias: LAUNCH_DARKLY_SDK_KEY
         - name: launch-darkly-offline-mode
           alias: LAUNCH_DARKLY_OFFLINE_MODE
+        - name: reports-email-username
```

Comment:
Can we call this payments-report and so forth? Just in case we add any more reports later on. It makes it clearer :)

Reply:
updated